### ISSUE TITLE
What does the Change Emails "With Issues" option mean under notification settings?

### ISSUE THREAD

**Issue by cswilliams** [Netflix/security_monkey#945]
Under notification settings, I set Change Emails to be "With Issues".  I assumed this meant it would only email me on changes with unjustified issues, however I still get emails for changes that have no issues at all (e.g. creating / deleting RDS snapshots, updating route 53 records, etc.).  Is this a bug or am I not understanding what "With issues" means?  Also, I am using the latest code in the develop branch.

Thanks!

**Comment by mikegrima** [comment]
Probably a bug

**Comment by cswilliams** [comment]
Yeah, I'm wondering if that whole notifications panel isn't working or something else is not working on my system.  I just set the change emails to "None" and restarted the supervisor (I also rebooted the system just to make sure there were no old processes still hanging around), however I still continue to get change emails.  Also, I have "Daily Email" checked and I receive no daily emails.  It's almost as if it's ignoring my choices in the notifications panel and using some set of defaults instead.
